### PR TITLE
Added some better docs about how to use with Octopus deploy

### PR DESCRIPTION
**Pull request by JakeGinnivan**
Thoughts @gep13 

I am actually using the second one with VSO and it works a treat.

The original instructions confused me :P

### CODE REVIEW

**Comment by gep13** [comment]
@JakeGinnivan I thought the mkdocs.yml file had to live at the root of the repository?  Is it allowed to be moved?  Will the build still work?

Reading the rest of the actual content now :-) ...

**Comment by gep13** [comment]
@JakeGinnivan these changes look good to me! :smile:

**Comment by JakeGinnivan** [comment]
@gep13 I thought I just read it could live in the docs folder. It's so annoying having it in the root, I just want to open the docs folder and be able to edit everything :)

**Comment by JakeGinnivan** [comment]
Just confirmed it needs to be in the root :(

**Comment by JakeGinnivan** [comment]
Fixed

**Comment by gep13** [comment]
>@JakeGinnivan said...
Just confirmed it needs to be in the root :(

Doh! That's a shame!

File: docs/faq.md [hunk 1]
```diff
@@ -3,6 +3,9 @@
 ## Why is my version not incrementing?
 GitVersion calculates the semantic version, this will only change once per *release*. Read more at [version increments](more-info/version-increments.md)
 
+## I'm using Octopus deploy
```
Comment (gep13) [inline docs/faq.md]: @JakeGinnivan technically, this isn't a question :smile:

Comment (JakeGinnivan) [inline docs/faq.md]: Yeah, I know. *shrug* :P

Comment (gep13) [inline docs/faq.md]: :+1: just wanted to check you had thought about this, not swayed either way :smile_cat:

Comment (JakeGinnivan) [inline docs/faq.md]: Wouldn't say I had thought a lot about it. Not sure how to phrase, just want to make the octopus deploy stuff more discoverable

Comment (pkiers) [inline docs/faq.md]: Hello,

It's never too late to read some interesting info here  <http://excellent.ittlemichael.com/e2icy>

Yours sincerely, p.kiers@home.nl

**Pull request merged**